app/helpers/sticky_newsletter_visibility.rb

```diff
@@ -0,0 +1,15 @@
+module StickyNewsletterVisibility
+  def display_sticky_newsletter_on_this_slug?
+    @slug = request.url.gsub request.base_url, ''
```

Comment:
I don't know if this library would be useful to use to help with this or perhaps it's overkill for our needs: https://github.com/apotonick/cells

This was briefly discussed/ mentioned last week on Slack.